prosessering-core/src/main/kotlin/no/nav/familie/prosessering/internal/TaskTransactionSynchronization.kt

```diff
@@ -0,0 +1,16 @@
+package no.nav.familie.prosessering.internal
+
+import org.slf4j.LoggerFactory
+import org.springframework.stereotype.Component
+import org.springframework.transaction.support.TransactionSynchronization
+
+@Component
+class TaskTransactionSynchronization(private val taskStepExecutorService: TaskStepExecutorService) :
+    TransactionSynchronization {
+
+    private val logger = LoggerFactory.getLogger(javaClass)
+    override fun afterCommit() {
+        logger.debug("Kaller på pollAndExecute")
+        taskStepExecutorService.pollAndExecute()
```

Comment:
Denne kaller ikke nødvendigvis på tasken du akkurat har lagret ned, men tar "neste på lista". Det er jo greit nok for de aller fleste tilfellene, men kan være litt misvisende at det heter `saveAndRun` - ettersom det egentlig er `saveAndPoll`. 

Eks: 
1. EF kjører g-omregning med X tusen tasker
2. En `saveAndRun` på oppdaterOppgave blir opprettet
3. Da kommer trolig ikke oppdaterOppgaveTasken til å bli kjørt "med en gang"

Reply:
Det er sant, har fikset i https://github.com/navikt/familie-prosessering-backend/pull/478/commits/1a2837f47eef750e56f2f662d324b358a2f5b7b5